Airports/EGOV/SMR/Labels.txt

```diff
@@ -0,0 +1,19 @@
+;Holding points
+"A1" N053.14.28.215 W004.31.10.239 standHold
+"A2" N053.14.37.811 W004.31.29.162 standHold
+"A3" N053.14.45.162 W004.31.43.536 standHold
+"A4" N053.15.06.100 W004.32.14.704 standHold
+"A5" N053.15.10.442 W004.32.24.830 standHold
+"A7" N053.15.15.924 W004.32.43.705 standHold
+"B1" N053.14.23.345 W004.31.15.918 standHold
+"B2" N053.14.33.344 W004.31.35.407 standHold
+"B3" N053.14.39.278 W004.31.52.185 standHold
+"B4" N053.14.50.365 W004.32.16.928 standHold
+"B5" N053.14.55.405 W004.32.27.014 standHold
+"B7" N053.15.11.463 W004.32.49.922 standHold
+"C1" N053.14.36.864 W004.32.20.439 standHold
+"C2" N053.14.37.643 W004.32.30.342 standHold
+"D1" N053.15.10.471 W004.32.15.529 standHold
+"D2" N053.15.25.735 W004.32.10.408 standHold
+"D3" N053.15.27.296 W004.32.21.308 standHold
+"E3" N053.14.49.129 W004.32.29.822 standHold
```

Comment:
```suggestion
"A1" N053.14.27.377 W004.31.08.165 standHold
"A2" N053.14.36.911 W004.31.26.788 standHold
"A3" N053.14.44.372 W004.31.41.549 standHold
"A4" N053.15.04.684 W004.32.16.036 standHold
"A5" N053.15.09.824 W004.32.25.692 standHold
"A7" N053.15.16.374 W004.32.44.867 standHold
"B1" N053.14.22.568 W004.31.14.787 standHold
"B2" N053.14.32.434 W004.31.33.823 standHold
"B3" N053.14.38.154 W004.31.49.688 standHold
"B4" N053.14.50.922 W004.32.16.036 standHold
"B5" N053.14.54.155 W004.32.28.313 standHold
"B7" N053.15.11.980 W004.32.50.523 standHold
"C1" N053.14.37.408 W004.32.20.174 standHold
"C2" N053.14.38.237 W004.32.29.968 standHold
"D1" N053.15.11.648 W004.32.15.070 standHold
"D2" N053.15.25.908 W004.32.09.552 standHold
"D3" N053.15.28.395 W004.32.22.105 standHold
"E3" N053.14.49.844 W004.32.29.968 standHold
```

I think this is more what @Robbo599 and I had in mind. So they're near the line rather than on it.

![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/1513e69a-abcb-4960-9671-a13889cb6605)